components/header.js

```diff
@@ -1,4 +1,3 @@
-import { useContext } from "react";
```

Comment:
the changes here, mainly revert #4's changes since we would just not show the header altogether when the extension is embedded